### PR TITLE
Loki ipv6 support

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
Looks to be working now (with a dev-scripts update https://github.com/openshift-metal3/dev-scripts/pull/1569 )
e.g. results [here](https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%7B%22datasource%22:%22PCEB727DF2F34084E%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22PCEB727DF2F34084E%22%7D,%22editorMode%22:%22code%22,%22expr%22:%22%7Binvoker%3D~%5C%22.%2A4%5C%5C%5C%5C.14-e2e-metal-ipi.%2A%5C%22,%20namespace%3D%5C%22openshift-machine-api%5C%22%7D%20%7C%20unpack%7C%20pod%20%3D~%20%60metal3-.%2A%60%20%7C%20container%20%3D~%20%5C%22metal3-static-ip-manager%5C%22%7C~%20%60ip%20addr%20add%60%22,%22queryType%22:%22range%22%7D%5D,%22range%22:%7B%22from%22:%221693299600000%22,%22to%22:%221693321199000%22%7D%7D)

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

File: ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh [hunk 1]
```diff
@@ -9,18 +9,23 @@ then
   exit 0
 fi
 
+
+PROXYLINE=
+if test -f "${SHARED_DIR}/proxy-conf.sh" && [[ "$JOB_NAME" =~ .*ipv6.* ]]
+then
+    # proxy-conf.sh holds the IPv4 address, we can't use it here
+    PROXYCFGLINE="        proxy_url: http://[fd00:1101::1]:8213"
```
[the remark this answers is not in the document]
Comment (derekhiggins) [reply ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:17]: > /tmp/yq w -i "${SHARED_DIR}/promtail.yml" 'clients[0].proxy_url' http://[fd00:1101::1]:8213

hmm, I havn't set values with yq before, looks like the correct command would be 
/tmp/yq -i '.clients.[0].proxyurl = "http://[fd00:1101::1]:8213"' /tmp/promtail.yml

I tried this out and hit problems with having the correct indent in the nested yaml, 

I think I'd prefer to leave it as is as I think its simpler and follows how some of the other preexisting values are being set in the yaml.

**Comment by dgoodwin** [comment]
/hold

Please release hold if you can confirm it's still working on a non-metal job.

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/unhold 
verified that logs appeared in loki for both rehearse-42406-periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 and rehearse-42406-periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn

@dgoodwin can you take a quick look again, I had to fix a env variable to set a default.

**Pull request merged**